public class FunctionUse12818 {

    public void funcUse() {
		org.apache.hadoop.io.SecureIOUtils.AlreadyExistsException alreadyexistsexception = new org.apache.hadoop.io.SecureIOUtils.AlreadyExistsException();
    }
}